wordpress-mobile/WordPress-Android
# Jetpack Focus: Jetpack feature card tidy UI

Parent #17336 

This PR makes minor adjustments to the Jetpack Feature Card UI
- Change Learn More link to Jetpack Green 50
- Change content text to make it darker

**Merge Instructions**
1 - ~~Ensure that #17681, #17692, and #17693 have been merged~~
2 - ~~Remove "Not ready for merge" label~~
3 - Merge as normal

Light | Dark
-- | --
<img width="250" height="500" alt="Light" src="https://user-images.githubusercontent.com/506707/208998606-69d86cbf-8f8a-44d9-8d0b-d2286d63b93b.png"> | <img width="250" height="500" alt="Dark" src="https://user-images.githubusercontent.com/506707/208998591-e2cc956b-9651-4bc1-80f7-05f96afbfeaa.png">

RTL Light | RTL Dark
-- | --
<img width="250" height="500" alt="Light" src="https://user-images.githubusercontent.com/506707/208998685-ebb48a33-c844-4787-b08b-ef5fa030452c.png"> | <img width="250" height="500" alt="Dark" src="https://user-images.githubusercontent.com/506707/208998682-ee0a4909-e2f9-4794-a6d3-6a9f16c54c78.png">

**To test:**
- Install WP app
- Login 
- Navigate to Me -> App Settings -> Debug Settings -> 
- Enable `jp_removal_three` and restart the app
- Navigate to My Site
- ✅ Verify that the Jetpack Feature card is shown and the view matches the screenshots
- Navigate to Me -> App Settings -> Change the Default appearance to light/dark (the opposite of what you just tested)
- Navigate back to the My Site menu tab
- ✅ Verify that the Jetpack Feature card is shown and the view matches the screenshots
- Navigate to Me -> App Settings and change the language to a RTL (I used Hebrew)
- Navigate back to the My Site menu tab
- Repeat the test steps above (**NOTE**: After changing the language, you should swipe the app closed and reopen to see the difference in the feature card - this is a quirk of Android - it has to fully remove the cardview from the backstack.)

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
